index.html

```diff
@@ -150,5 +150,28 @@ <h2 class="list-of-skills">Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact-section">
+    <div class="contact-details">
+      <h2 class="paragraph_1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+    <form class="form" action="">
+      <ul>
+        <li><input type="text" placeholder="Full Name"></li><hr>
+        <li><input type="email" placeholder="Email address"></li><hr>
+        <li><textarea name="Comment" placeholder="Enter text here" rows="5"></textarea></li>
+      </ul>
+      <button class="submit">Get in touch</button>
```

Comment:
:red_circle:  Kindly make sure you add the type attribute to your button.The type of your button should be `submit` as you have to submit (post) the form data to the action URL.